samples/suit/recovery/README.rst

```diff
@@ -10,41 +10,40 @@ SUIT recovery application
    :depth: 2
 
 The SUIT recovery application is a minimal application that allows recovering the device firmware if the original firmware is damaged.
-It is to be used as a additional companion firmware to the main application using SUIT rather than a standalone application.
+It is to be used as a companion firmware to the main application using SUIT rather than a standalone application.
 
-The application uses the SMP protocol via BLE and SUIT to perform the recovery.
+The application uses the SMP protocol via Bluetooth LE and SUIT to perform the recovery.
+The main application which uses this recovery firmware must also be using Bluetooth LE.
```

Comment:
Is this really a requirement?  I think I am able to run smp_transfer in UART configuration with BLE recover app just fine.

Reply:
OK, I will delete this sequence if it causes controversy. I wasn't sure if some hardware configuration through UICR wasn't needed, that's why I wanted to stay safe.